code/game/gamemodes/dynamic/dynamic.dm

```diff
@@ -116,13 +116,13 @@ GLOBAL_VAR_INIT(dynamic_forced_threat_level, -1)
 	/// Equivalent to threat_curve_centre, but for the budget split.
 	/// A negative value will weigh towards midround rulesets, and a positive
 	/// value will weight towards roundstart ones.
-	var/roundstart_split_curve_centre = 1
+	var/roundstart_split_curve_centre = 2.5
```

Comment:
This would need to be tested, currently it's anyone's guess how this would actually affect the rulesets picked.

Reply:
This was specifically requested by Zender- He wants roundstart roles to take priority